resources/aws_ec2.go

```diff
@@ -541,6 +544,14 @@ func (obj *AwsEc2Res) CheckApply(apply bool) (checkOK bool, err error) {
 		}
 		runParams.SetMinCount(1)
 		runParams.SetMaxCount(1)
+		if obj.UserData != "" {
+			userDataBytes, err := ioutil.ReadFile(obj.UserData)
```

Comment:
Is there a reason why this is a filepath instead of the actual string contents?

Reply:
@purpleidea I think the most likely use case is to have a bash script saved to a file, rather than inline, no?